tests/TestData/Providers/ReflectionTestDataProviders.php

```diff
@@ -4,38 +4,159 @@
 namespace StubTests\TestData\Providers;
 
 use Generator;
+use StubTests\Model\StubProblemType;
 
 class ReflectionTestDataProviders
 {
 
     public static function constantProvider(): ?Generator
     {
         foreach (ReflectionStubsSingleton::getReflectionStubs()->getConstants() as $constant) {
-            yield "constant {$constant->name}" => [$constant];
+            if (!$constant->hasMutedProblem(StubProblemType::STUB_IS_MISSED)) {
+                yield "constant {$constant->name}" => [$constant];
+            }
+        }
+    }
+
+    public static function constantValuesProvider(): ?Generator
+    {
+        foreach (ReflectionStubsSingleton::getReflectionStubs()->getConstants() as $constant) {
+            if (!$constant->hasMutedProblem(StubProblemType::STUB_IS_MISSED) &&
+                !$constant->hasMutedProblem(StubProblemType::WRONG_CONSTANT_VALUE)) {
+                yield "constant {$constant->name}" => [$constant];
+            }
+        }
+    }
+
+    public static function classConstantProvider(): ?Generator
+    {
+        foreach (ReflectionStubsSingleton::getReflectionStubs()->getClasses() as $class) {
```

Comment:
I don't like the duplication in this place, let's combine `getClasses` and `getInterfaces` into a single method and iterate over it. Or we can create an array that includes results of both calls and iterate over it.